Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -146,11 +150,18 @@ cannot be guaranteed to be immutable.
 The listeners
 -------------
 
-Extensions and PHP packages can add listeners that are registered via YAML. They
+Extensions and PHP packages can add listeners that are registered via YAML (or starting
+with TYPO3 v13 also natively in :ref:`PHP using attributes <EventDispatcherImplementation>`). They
 are usually associated to :php:`Event` objects by the fully-qualified class name
 of the event to be listened on. It is the task of the listener provider to
 provide configuration mechanisms to represent this relationship.
 
+If multiple event listeners for a specific event are registered, their order can
+be configured or an event listener can also be overridden with a different one.
```

Comment:
I suggest to write "or an existing event listener" to emphasize that.

Reply:
Good idea, changed.